app/assets/templates/header.html

```diff
@@ -1,10 +1,10 @@
-<div class="background container-fluid">
+<div class="container-fluid">
   <div class="row">
     <div class="header-bar col-xs-10 col-xs-offset-1">
       
       <div class="header-left col-xs-10">
         <div class="row">
-          <a href="#" class="lineup btn btn-default col-xs-1" id="logo"><img src="assets/logo.gif"  height="20px"></a>
+          <a href="#/" class="lineup btn btn-default col-xs-1" id="logo"><img src="assets/logo.gif"  height="20px"></a>
           <input type="text" id="search-bar" class="btn btn-default lineup col-xs-9" placeholder="Search">
           <button id="dropdown" class="lineup btn btn-default dropdown-toggle col-xs-1" data-toggle="dropdown">
             <div class="glyphicon glyphicon-list"></div>
```

Comment:
Yes wor... 
that's the logo that goes back to the root file, "Home" page...
local host works wor...